frontend/src/api/k8s/inferenceServices.ts

```diff
@@ -79,6 +81,8 @@ export const assembleInferenceService = (
                 key: dataConnectionKey,
                 path,
               },
+              args: servingRuntimeArgs,
+              env: servingRuntimeEnvVars,
```

Comment:
Either here or before we get here, we should filter out any empty strings from `servingRuntimeArgs` and filter out any env vars with an empty string for the `name`. Then we'll need to update the tests/mocks changed by this PR so they have `args: []` instead of `args: [""]` as the empty value.